src/Resources/views/admin/Content/_form.html.twig

```diff
@@ -0,0 +1,66 @@
+{% form_theme form with [
+    '@SyliusAdmin/Form/theme.html.twig',
+    '@SherlockodeAdvancedContent/Form/content.html.twig',
+    '@SherlockodeAdvancedContent/Form/translations.html.twig'
+] %}
+
+{% set content = form.vars.data %}
+{{ form_errors(form) }}
+<div class="ui grid">
+    <div class="twelve wide column">
+        <div class="ui fluid styled accordion">
+            <div class="active title">
+                <i class="dropdown icon"></i>
+                {{ 'sherlockode_sylius_acb.form.general'|trans }}
+            </div>
+            <div class="active content">
+                {{ form_row(form.name) }}
+                {{ form_row(form.slug) }}
+            </div>
+        </div>
+        <div class="ui fluid styled accordion">
```

Comment:
We can add a `<div class="ui hidden divider"></div>` before this line, to increase space between this block and the previous one.